ledger-core/virtual/execute/execute.go

```diff
@@ -9,7 +9,6 @@ package execute
 
 import (
 	"context"
-
```

Comment:
this is probably not intentional

Reply:
definitely it is not ... some random tap?

Reply:
fixed